ca/ca.go

```diff
@@ -492,8 +493,24 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(ctx context.Context, csr x5
 		return emptyCert, err
 	}
 
+	var ocspResp []byte
+	if features.Enabled(features.GenerateOCSPEarly) {
+		ocspResp, err = ca.GenerateOCSP(ctx, core.OCSPSigningRequest{
+			CertDER: certDER,
+			Status:  "good",
+		})
+		if err != nil {
+			err = berrors.InternalServerError(err.Error())
+			ca.log.AuditInfo(fmt.Sprintf("OCSP Signing failure: serial=[%s] pem=[%s] err=[%s]",
+				serialHex, certPEM, err))
+			// If OCSP generation fails don't return an error and orphan the
```

Comment:
I think as worded it's not clear whether the comment is saying "We will orphan the cert here," or "We do X to avoid orphaning the cert."

For instance you could say: "Ignore errors here to avoid orphaning the certificate. The ocsp-updater will attempt to update the OCSP status when it sees it missing."